### ISSUE TITLE
version okio-2.10.0 does not get dependencies correctly

### ISSUE THREAD

**Issue by GauravDeshmukhGW** [square/okio#954]
With okio-2.10.0 when using the Gradle, the files are incorrectly named.
The library pom file looks like this:
```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <!-- This module was also published with a richer model, Gradle metadata,  -->
  <!-- which should be used instead. Do not delete the following line which  -->
  <!-- is to indicate to Gradle or any Gradle module metadata file consumer  -->
  <!-- that they should prefer consuming it instead. -->
  <!-- do_not_remove: published-with-gradle-metadata -->
  <modelVersion>4.0.0</modelVersion>
  <groupId>com.squareup.okio</groupId>
  <artifactId>okio</artifactId>
  <version>2.10.0</version>
  <dependencies>
    <dependency>
      <groupId>org.jetbrains.kotlin</groupId>
      <artifactId>kotlin-stdlib</artifactId>
      <version>1.4.20</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>org.jetbrains.kotlin</groupId>
      <artifactId>kotlin-stdlib-common</artifactId>
      <version>1.4.20</version>
      <scope>compile</scope>
    </dependency>
  </dependencies>
  <description>A modern I/O API for Java</description>
  <name>Okio</name>
  <url>https://github.com/square/okio/</url>
  <licenses>
    <license>
      <name>The Apache Software License, Version 2.0</name>
      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
  <scm>
    <url>https://github.com/square/okio/</url>
    <connection>scm:git:git://github.com/square/okio.git</connection>
    <developerConnection>scm:git:ssh://git@github.com/square/okio.git</developerConnection>
  </scm>
  <developers>
    <developer>
      <id>square</id>
      <name>Square, Inc.</name>
    </developer>
  </developers>
</project>
```

And the downloaded files looks like:
<img width="975" alt="Screen Shot 2021-06-09 at 3 22 19 PM" src="https://user-images.githubusercontent.com/64865171/121437117-9929c300-c936-11eb-9180-f031b2fdfdbc.png">

As you can see in the screenshot attached, the downloaded files listed a jar file `okio-jvm-2.10.0` whereas the pom listed `okio-2.10.0` this causes the Gradle to throw the following error:
```
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':runStudio'.
> Could not resolve all files for configuration ':detachedConfiguration1'.
   > Could not find okio-2.10.0.jar (com.squareup.okio:okio:2.10.0).
```

Please look into this issue, as this is causing build failures on our systems. Thank you.

**Comment by 3ll3d00d** [comment]
I just noticed the same in 2.8.0 (pulled in via okhttp3)

the gradle module metadata has the following in the jvm-api and jvm-runtime variants, i.e. name != url

```
"files": [
        {
          "name": "okio-jvm-2.8.0.jar",
          "url": "okio-2.8.0.jar",
          "size": 243179,
          "sha512": "41f347b5e111011f399f7467225e0c8ab94e94f4354537ad1a4d23c6a686ee88eecf4425f59e59a9cdf55aebfb8a26636557d0038a5bb8ceea064a6cf29fe1bf",
          "sha256": "4496b06e73982fcdd8a5393f46e5df2ce2fa4465df5895454cac68a32f09bbc8",
          "sha1": "49b64e09d81c0cc84b267edd0c2fd7df5a64c78c",
          "md5": "48756cee44c4f8403ebd07a8152ea5ba"
        }
      ]
```

**Comment by DaxInvader** [comment]
Same problem on my end. what would be a temporary fix?

**Comment by swankjesse** [comment]
I believe this is fixed in the 3.0.0 alpha releases.
